### DISCUSSION TITLE
Health of Curiefense incubation project #1192 (vote closed)

### DISCUSSION THREAD

**Discussion by git-vote** [cncf/toc#1360] [Announcements]
The vote for "**Health of Curiefense incubation project** (**#1192**)" is now closed.

## Vote results

The vote **passed**! 🎉

`72.73%` of the users with binding vote were in favor (passing threshold: `66%`).

### Summary

|        In favor        |        Against        |       Abstain        |        Not voted        |
| :--------------------: | :-------------------: | :------------------: | :---------------------: |
| 8 | 0 | 0 | 3 |

### Binding votes (8)
| User | Vote  | Timestamp |
| ---- | :---: | :-------: |
| @TheFoxAtWork | In favor | 2024-06-25 0:36:17.0 +00:00:00 |
| @rochaporto | In favor | 2024-06-27 16:59:50.0 +00:00:00 |
| @kevin-wangzefeng | In favor | 2024-06-26 3:45:21.0 +00:00:00 |
| @kgamanji | In favor | 2024-06-26 8:38:01.0 +00:00:00 |
| @nikhita | In favor | 2024-06-25 6:03:16.0 +00:00:00 |
| @dims | In favor | 2024-06-25 1:10:09.0 +00:00:00 |
| @linsun | In favor | 2024-06-25 22:37:00.0 +00:00:00 |
| @angellk | In favor | 2024-06-25 0:52:10.0 +00:00:00 |

**Comment by AKJUS** [comment]
keep in row but don't try to be in 1st...ak
